independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/update/Operation.java

```diff
@@ -0,0 +1,9 @@
+package io.quarkus.devtools.project.update;
+
+import io.quarkus.devtools.project.BuildTool;
+
+import java.util.Map;
+
+public interface Operation {
+    Map<String, Object> toMap(BuildTool buildTool);
```

Comment:
Can you add some javadocs or use less generic names?

Reply:
Fixed